Description: Silly documentation fix that prevents 2nd build in a raw.
Author: Kartik Mistry <kartik@debian.org>
Last-Update: 2012-04-20

--- dee-1.0.10.orig/doc/reference/dee-1.0/tmpl/dee-server.sgml
+++ dee-1.0.10/doc/reference/dee-1.0/tmpl/dee-server.sgml
@@ -9,13 +9,11 @@ DeeServer
 
 </para>
 
-
 <!-- ##### SECTION See_Also ##### -->
 <para>
 
 </para>
 
-
 <!-- ##### SECTION Stability_Level ##### -->
 
 
--- dee-1.0.10.orig/doc/reference/dee-1.0/tmpl/dee-client.sgml
+++ dee-1.0.10/doc/reference/dee-1.0/tmpl/dee-client.sgml
@@ -9,13 +9,11 @@ DeeClient
 
 </para>
 
-
 <!-- ##### SECTION See_Also ##### -->
 <para>
 
 </para>
 
-
 <!-- ##### SECTION Stability_Level ##### -->
 
 
--- dee-1.0.10.orig/doc/reference/dee-1.0/tmpl/dee-icu.sgml
+++ dee-1.0.10/doc/reference/dee-1.0/tmpl/dee-icu.sgml
@@ -9,13 +9,11 @@ dee-icu
 
 </para>
 
-
 <!-- ##### SECTION See_Also ##### -->
 <para>
 
 </para>
 
-
 <!-- ##### SECTION Stability_Level ##### -->
 
 
